universal-login-sdk/test/helpers/SetupSdk.ts

```diff
@@ -0,0 +1,12 @@
+const relayerUnderTest = require('universal-login-relayer/build/utils/relayerUnderTest').default;
```

Comment:
Next PR:
* add types to Realyer and RelayerUnderTest
* use import relayerUnderTest from 'universal-login-relayer/build/utils/relayerUnderTest';